python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -577,8 +582,9 @@ class RandomForestRegressor(Base):
         cdef uintptr_t X_ptr, y_ptr
 
         X_m, n_rows, self.n_cols, self.dtype = \
-            input_to_cuml_array(X, check_dtype=[np.float32, np.float64],
-                                order='F')
+            input_to_cuml_array(
```

Comment:
What do you think about _always_ warning if a user fits on float64? We'd warn that this model cannot be pickled and would use cpu-based prediction. That may encourage more users to move to float32. I really don't think anyone should use float64 for training to be honest - it leads to lots of limitations and seems _very_ unlikely to improve accuracy since we use quantized values for training anyways.

Reply:
Currently we do throw a warning when a user fits on float64. However, the warning only includes not being able to use GPU predict. Will update the warning to include pickling as well..